RxSwift/Traits/PrimitiveSequence/Completable.swift

```diff
@@ -123,12 +123,7 @@ extension PrimitiveSequenceType where Trait == CompletableTrait, Element == Swif
                 let callStack = [String]()
         #endif
 
-        let disposable: Disposable
-        if let onDisposed = onDisposed {
-            disposable = Disposables.create(with: onDisposed)
-        } else {
-            disposable = Disposables.create()
-        }
+        let disposable: Disposable = onDisposed.map({ Disposables.create(with: $0) }) ?? Disposables.create()
```

Comment:
```suggestion
        let disposable: Disposable = onDisposed.map(Disposables.create(with:)) ?? Disposables.create()
```

Reply:
Clever suggestion! I committed your suggestion, and will commit the others as well.